tutorials/tts/NeMo_TTS_Primer.ipynb

```diff
@@ -129,7 +129,7 @@
     "While this is the most common structure, there may be fewer or additional steps depending on the use case. For example, some languages do not require G2P and can instead rely on the model to convert raw text/graphemes to spectrogram.\n",
     "\n",
     "<div>\n",
-    "<img src=\"https://raw.githubusercontent.com/NVIDIA/NeMo/main/tutorials/tts/images/tts_pipeline.png\" width=\"500\", height=\"500\"/>\n",
+    "<img src=\"https://raw.githubusercontent.com/NVIDIA/NeMo/stable/tutorials/tts/images/tts_pipeline.png\" width=\"500\", height=\"500\"/>\n",
```

Comment:
One question about which branch should be applied cause I am not sure about our convention:
1. choose `main` or `stable` for `main` branch tutorials?
2. choose `release version` for `release` branch tutorials?

Reply:
discussed offline. links should always be `stable` no matter it is `main` or `release` branch.

Reply:
Hmm thats not the case. Main branch notebooks should have main as their branch, release branch should have stable as their branch when selecting version to install inside the notebook. When release branch is cut, this is changed from "main" to "rABC.XYZ" 

See https://github.com/NVIDIA/NeMo/blob/r1.16.0/tutorials/00_NeMo_Primer.ipynb and https://github.com/NVIDIA/NeMo/blob/main/tutorials/00_NeMo_Primer.ipynb

Reply:
hmm...I was advised that links should always be `stable`. @ericharper may provide a better explanation here.